.pre-commit-config.yaml

```diff
@@ -2,22 +2,28 @@ repos:
 -   repo: https://github.com/PyCQA/pylint
     rev: pylint-2.6.0
     hooks:
-    -    id: pylint
-         language: system
-         entry: dotenv run pylint
-         verbose: true
-         args: ["--disable=W0511"]
+    -   id: pylint
+        language: system
+        entry: dotenv run pylint
+        verbose: true
+        args: ["--disable=W0511"]
+-   repo: https://github.com/psf/black
+    rev: 20.8b1
+    hooks:
+    -   id: black
 -   repo: https://github.com/pre-commit/mirrors-eslint
-    rev: v7.12.0
+    rev: v7.12.1
     hooks:
-    -    id: eslint
-         verbose: true
-         args: ["--max-warnings=0"]
+    -   id: eslint
+        language: system
+        entry: frontend/node_modules/.bin/eslint 
```

Comment:
Unable to run eslint on windowns.

On Windows eslint.cmd needs to be run.

For now replave it with bash.